db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
```

Comment:
I'm not a big fan of database validations. They explode pretty hard. I know there are differing schools of thought, but I find AR validations much easier to deal with and just as powerful for anything we'd likely need here. The timestamps one was just in there.